RafRate.Data/Contexts/ConnectionStringContext.cs

```diff
@@ -2,17 +2,21 @@
 
 public class ConnectionStringContext
 {
-    // TODO don't work 
-    private string _connectionString;
-
-    ConnectionStringContext()
+    public ConnectionStringContext()
     {
-        _connectionString = Environment.GetEnvironmentVariable("ConnectionString") 
-                            ?? throw new InvalidOperationException();
+        _connectionString = Environment.GetEnvironmentVariable("ConnectionString")
+                           ?? throw new InvalidOperationException();
     }
-
-    public string ReturnConnectionString()
+    private string _connectionString;
+    
+    public string ConnectionString
```

Comment:
You can use
public string ConnecionString { get; set; }

Reply:
also use static property and not need to create this object in RafRate.API/Configuration/Extentions/DataBaseExtension.cs - line 8